default.nix

```diff
@@ -0,0 +1,20 @@
+{ stdenv
+, pkgs ? (import <nixpkgs> {})
+}:
+
+let
+  env = with pkgs; [
+    bundler
```

Comment:
They are for entering a environment where the dependencies are available. This environment can be made pure, so no system packages pollute the build. With this I can do `nix-shell --run "bundler exec nanoc && bundler exec nanoc view"` and I can be sure that I can build this very commit of the site on another machine and resulting in the _exact_ same version, because the compiler and the whole environment around it is the very same. Also, it is an convenient thing for NixOS guys.

If you don't want it in the repo, I can understand that!